src/Servers/Kestrel/test/Sockets.FunctionalTests/SocketTranspotTests.cs

```diff
@@ -1,15 +1,22 @@
+using System.Buffers;
+using System.Diagnostics;
 using System.Net;
 using System.Net.Http;
 using System.Net.Sockets;
+using System.Text;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Connections.Features;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
 using Microsoft.AspNetCore.Server.Kestrel.FunctionalTests;
 using Microsoft.AspNetCore.Testing;
 using Microsoft.Extensions.Hosting;
 using Xunit;
 
+using KestrelHttpVersion = Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpVersion;
+using KestrelHttpMethod = Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpMethod;
+
 namespace Sockets.FunctionalTests
 {
     public class SocketTranspotTests : LoggedTestBase
```

Comment:
Not introduced in this change, but file and type are misnamed.  Should be SocketTranspo<strong><em>r</em></strong>tTests .